pkg/engine/generation.go

```diff
@@ -70,6 +70,7 @@ func filterRules(policy kyverno.ClusterPolicy, resource unstructured.Unstructure
 	for _, rule := range policy.Spec.Rules {
 		if ruleResp := filterRule(rule, resource, admissionInfo, ctx, log); ruleResp != nil {
 			resp.PolicyResponse.Rules = append(resp.PolicyResponse.Rules, *ruleResp)
+			resp.Synchronize = rule.Generation.Synchronize
```

Comment:
Check it carefully. Does it Break anything ?

Reply:
How would this impact the existing generate policy? The old generate policy does not have this flag, we should considering handle those policies.

Reply:
@realshuting  Only one case I am assuming that is if we get two ruleResp then it will overwrite the Synchronize

Reply:
Rule and GenerateRequest is one to one map, why will it be overwritten?